vclib/src/commonTest/kotlin/at/asitplus/wallet/lib/iso/CborSerializationTest.kt

```diff
@@ -24,6 +26,10 @@ import kotlinx.serialization.encodeToString
 
 class CborSerializationTest : FreeSpec({
 
+    beforeSpec {
+        Napier.base(DebugAntilog())
```

Comment:
Really needed here?

Reply:
should be moved to global test configuration, yes!

Reply:
if you only execute a single test, napier could be uninitalized.